### ISSUE TITLE
Not sure if we want prefix variants like `prefix_math.sin x` in addition to `x.sin`

### ISSUE THREAD

**Issue by michaellilltokiwa** [tokiwa-software/fuzion#4040]
Not sure if we want prefix variants like `prefix_math.sin x` in addition to `x.sin`. Maybe keep this as a new issue after #2362 was closed.

_Originally posted by @fridis in https://github.com/tokiwa-software/fuzion/pull/4037#pullrequestreview-2401428970_
